src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Generated/FrameworkContentElement.cs

```diff
@@ -1227,6 +1227,7 @@ internal bool PotentiallyHasMentees
         #region Internal Fields
 
         // Optimization, to avoid calling FromSystemType too often
+        // todo BSC: type
         internal new static DependencyObjectType DType = DependencyObjectType.FromSystemTypeInternal(typeof(FrameworkContentElement));
```

Comment:
What does this mean ??

Reply:
BSC is short for "Bastian Schmidt".
I left the todo there because i couldn't find any usages of the `DType` fields.
Should it be removed, or is it used through reflection somewhere?

Reply:
With the changes in your PR, this type is no more used. So I think, we can go ahead and remove the `DType` field.